roles/ocp/tasks/process.yml

```diff
@@ -13,6 +13,9 @@
                       {%- else -%}
                       {{ openshift_channel }}
                       {%- endif -%}"
+  environment: 
+    AWS_PROFILE: 'rhacm-automation'
+    GOOGLE_APPLICATION_CREDENTIALS: "{{ lookup('env', 'HOME') }}/.{{ creds_path }}/{{ creds_file }}"
```

Comment:
The {{ creds_path }}/{{creds_file }} will not work here, since those variables set as part of another loop.
Please, test your changes before submitting the patch to catch such errors.